src/journal-remote/journal-remote-main.c

```diff
@@ -37,6 +38,7 @@ static const char *arg_getter = NULL;
 static const char *arg_listen_raw = NULL;
 static const char *arg_listen_http = NULL;
 static const char *arg_listen_https = NULL;
+static CompressionArgs *arg_compression = NULL;
```

Comment:
Ah, sorry, my bad. Please do not make it pointer, but revert to the previous one:
```c
static CompressionArgs arg_compression = {};
```
Then, use
```
STATIC_DESTRUCTOR_REGISTER(arg_compression, compression_args_clear);
```
That is, please drop `p`. That's really my fault. Sorry.

So, `DEFINE_TRIVIAL_CLEANUP_FUNC(CompressionArgs*, compression_args_clear)` in the header should not be necessary anymore.